helm_deploy/templates/deployment.yaml

```diff
@@ -28,6 +28,22 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
+        containers:
+          # https://github.com/ministryofjustice/hmpps-utility-container-images
+        - name: clamav
+          image: ghcr.io/ministryofjustice/hmpps-clamav:latest
```

Comment:
apply encountered unexpected issues when using :latest. We there test and use the [latest verified sha](https://github.com/ministryofjustice/hmpps-utility-container-images/pkgs/container/hmpps-clamav/versions?filters%5Bversion_type%5D=tagged), manually bumping 

```
image: ghcr.io/ministryofjustice/hmpps-clamav:sha-e2459aa
```

Reply:
Looking at the packages latest has been the same sha for a while now and testing has not presented any problems. I will keep as is unless problems present themselves